scripts/commands/checkmission.lua

```diff
@@ -13,7 +13,20 @@ cmdprops =
 
 function onTrigger(player,logId,target)
     
+    local logName;
     logId = tonumber(logId) or _G[logId];
+    if ((type(logId) == "table")) then
+        logName = logId.full_name;
+        logId = logId.mission_log;
+    else
+        if (logId <= 2) then
+            local logNames = {"San d'Oria", "Bastok", "Windurst"}
```

Comment:
maaaybe
but commit log says i did that waaay before this was merged https://github.com/DarkstarProject/darkstar/commit/a8969109828a88c5455a0de3ae1c8c4b0a44ff12

changing it so its like how teo said should work fine though